channels/rdp2tcp/client/rdp2tcp_main.c

```diff
@@ -84,9 +84,9 @@ static int init_external_addin(Plugin* plugin)
 	}
 
 	// Execute plugin
-	if (!CreateProcess(NULL,
-	                   plugin->channelEntryPoints.pExtendedData, // command line
-	                   NULL,                                     // process security attributes
+	if (!CreateProcess(plugin->channelEntryPoints.pExtendedData,
```

Comment:
Not an expert with this code but is this switch intended? With the new code it is not possible to provide a command line with arguments to CreateProcess (as was before)

Reply:
https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessa

Both are valid and the first argument has the correct const parameters we require.

Reply:
Just tested on Win10 20H2
```
	WCHAR buf[] = { L"notepad.exe test.txt" };
	auto res = CreateProcessW(buf, nullptr, nullptr, nullptr, FALSE, 0, nullptr, nullptr, &suInfo, &procInfo);
```
Result: `FALSE` (GetLastError: `ERROR_FILE_NOT_FOUND`)

```
	WCHAR buf[] = { L"notepad.exe test.txt" };
	auto res = CreateProcessW(nullptr, buf, nullptr, nullptr, FALSE, 0, nullptr, nullptr, &suInfo, &procInfo);
```
Result: `TRUE` - notepad is started with test.txt as an argument